spyder/plugins/editor/widgets/codeeditor/multicursor_mixin.py

```diff
@@ -0,0 +1,460 @@
+            if skip:
+                # text folding swallows most input to prevent typing on folded
+                #    lines.
+                pass
+            # ---- handle Tab
+            elif key == Qt.Key.Key_Tab and not ctrl:  # ctrl-tab is shortcut
+                # Don't do intelligent tab with multi-cursor to skip
+                #   calls to do_completion. Avoiding completions with multi
+                #   cursor is much easier than solving all the edge cases.
+
+                self.indent(force=self.tab_mode)
+            elif key == Qt.Key.Key_Backtab and not ctrl:
+                increasing_position = False
+                # TODO ignore indent level of neighboring lines and simply
+                #    indent by 1 level at a time. Cursor update order can
+                #    make this unpredictable otherwise.
+                self.unindent(force=self.tab_mode)
+            # ---- handle enter/return
+            elif key in (Qt.Key_Enter, Qt.Key_Return):
+                if not shift and not ctrl:
+                    if (
+                        self.add_colons_enabled and
+                        self.is_python_like() and
+                        self.autoinsert_colons()
+                    ):
+                        self.insert_text(':' + self.get_line_separator())
+                        if self.strip_trailing_spaces_on_modify:
+                            self.fix_and_strip_indent()
+                        else:
+                            self.fix_indent()
+                    else:
+                        cur_indent = self.get_block_indentation(
+                            self.textCursor().blockNumber())
+                        self._handle_keypress_event(event)
+                        # Check if we're in a comment or a string at the
+                        # current position
+                        cmt_or_str_cursor = self.in_comment_or_string()
+
+                        # Check if the line start with a comment or string
+                        cursor = self.textCursor()
+                        cursor.setPosition(cursor.block().position(),
+                                           QTextCursor.KeepAnchor)
+                        cmt_or_str_line_begin = self.in_comment_or_string(
+                            cursor=cursor)
+
+                        # Check if we are in a comment or a string
+                        cmt_or_str = cmt_or_str_cursor and \
+                            cmt_or_str_line_begin
+
+                        if self.strip_trailing_spaces_on_modify:
+                            self.fix_and_strip_indent(
+                                comment_or_string=cmt_or_str,
+                                cur_indent=cur_indent)
+                        else:
+                            self.fix_indent(comment_or_string=cmt_or_str,
+                                            cur_indent=cur_indent)
+            # ---- intelligent backspace handling
```

Comment:
```suggestion
                            self.fix_indent(
                                comment_or_string=cmt_or_str,
                                cur_indent=cur_indent
                            )
            # ---- Intelligent backspace handling
```

Improve code style and fix capitalization.